ppl/archive/import.go

```diff
@@ -29,7 +29,10 @@ var (
 )
 
 // For unit testing.
-var importLZ4BlockSize = zngio.DefaultLZ4BlockSize
+var (
+	importLZ4BlockSize         = zngio.DefaultLZ4BlockSize
+	importDefaultStaleDuration = time.Second * 5
```

Comment:
Nit: Make importDefaultStaleDuration const, or sort us if you don't.

Reply:
const it is